pkgs/applications/version-management/git-and-tools/stgit/default.nix

```diff
@@ -23,7 +23,7 @@ python3Packages.buildPythonApplication rec {
 
   meta = with lib; {
     description = "A patch manager implemented on top of Git";
-    homepage = "http://procode.org/stgit/";
+    homepage = "https://stacked-git.github.io/";
     license = licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
It should be gpl2Only, according the header [here](https://github.com/stacked-git/stgit/blob/master/stg#L8-L10)

Happy to add another commit for this, if you think it's worthwhile.